core/tests/protractor/richTextComponents.js

```diff
@@ -83,6 +84,7 @@ describe('rich-text components', function() {
       // TODO (Jacob) Remove when
       // https://code.google.com/p/google-cast-sdk/issues/detail?id=309 is fixed
       'cast_sender.js - Failed to load resource: net::ERR_FAILED',
+      'chrome-extension://invalid/ - Failed to load resource: net::ERR_FAILED',
```

Comment:
Yeah, I think this is fine for now, as I really can't tell what the actual issue is. It might become apparent later if we notice it causing problems outside the context of travis tests. When Sean comes back he may have a better idea of what the cause is.

But yeah can you add a todo note above it?

Reply:
Done.